app/serializers/application.js

```diff
@@ -0,0 +1,23 @@
+import DS from 'ember-data';
+import Ember from 'ember';
+
+export default DS.JSONAPISerializer.extend({
+    keyForAttribute: function(attr) {
+        return Ember.String.underscore(attr);
+    },
+   keyForRelationship(key) {
```

Comment:
Some inconsistent spacing. Looks like you need to add a space from here down to the closing )

Reply:
thanks @jamescdavis, fixed.